src/fiatconnect/saga.test.ts

```diff
@@ -609,33 +607,13 @@ describe('Fiatconnect saga', () => {
         .put(attemptReturnUserFlowCompleted())
         .run()
       expect(navigate).toHaveBeenCalledWith(Screens.SelectProvider, selectProviderParams)
+      expect(Logger.debug).toHaveBeenCalledWith(
+        'FiatConnectSaga',
+        'Failed to use previous fiatAccount to take user directly to Review Screen',
+        new Error('Could not find quote')
+      )
     })
-    it('navigates to SelectProvider when the quote does not match the fiatAccountType', async () => {
```

Comment:
this was actually not testing this scenario, since this logic is in `_getSpecificQuote` and it was mocked. The test was passing because of an undefined access error was thrown